gist/gist.py

```diff
@@ -128,16 +129,18 @@ class GistAPI(object):
     This class defines the interface to github.
     """
 
-    def __init__(self, token, editor=None):
+    def __init__(self, token, editor=None, as_module=False):
```

Comment:
Added an argument that specifies how the class is being used is not a solution that I like. The class should behave the same in any environment. Create a ticket that describes the problem you are having here, and together we should be able to figure out an acceptable solution.